Node/src/TeamsMessage.ts

```diff
@@ -41,19 +41,54 @@ export enum MentionTextLocation {
   AppendText
 }
 
+export class MentionEntity {
+  type: string;
+  mentioned: any;
+  text: string;
+}
+
+export class UserMention extends MentionEntity
+{
+    constructor(userId: string, name: string) {
```

Comment:
I think the parameters should be (user: builder.IIdentity, text?: string), like the original method.